nuxeo-features/nuxeo-platform-userworkspace/nuxeo-platform-userworkspace-core/src/test/java/org/nuxeo/ecm/platform/userworkspace/core/tests/TestFavoritesUserWorkspace.java

```diff
@@ -74,5 +74,19 @@ public void testFavoritesWithoutDomain() {
         // no user favorites, always false
         assertFalse(favoritesManager.isFavorite(testFile, session));
         assertFalse(favoritesManager.isFavorite(testWorkspace, session));
+
+        try {
+            favoritesManager.addToFavorites(testFile, session);
+            fail("Should have raised DocumentNotFoundException");
+        } catch (DocumentNotFoundException e) {
+            assertEquals("No user favorites found", e.getMessage());
+        }
+
+        try {
+            favoritesManager.removeFromFavorites(testFile, session);
```

Comment:
You mean `testWorkspace`?

Reply:
No, I mean `testFile`, could work also with `testWorkspace` anyway as it always throws the exception :)